orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/TestTaskResolver.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.q.handler
+
+import com.netflix.spinnaker.orca.Task
+import com.netflix.spinnaker.orca.TaskResolver
+
+
+class TestTaskResolver : TaskResolver(emptyList()) {
+  override fun getTaskClass(taskTypeIdentifier: String): Class<out Task> {
+    return Class.forName(taskTypeIdentifier) as Class<out Task>
```

Comment:
I'd actually probably just pull this up to the `TaskResolver` as a catch-all fallback to the original behavior.

The test cases are a little different because they're just using interfaces rather than actual classes, thus the `Class.forName()` is needed.